epub33/a11y/index.html

```diff
@@ -50,6 +50,7 @@
 					repoURL: "https://github.com/w3c/epub-specs",
 					branch: "main"
 				},
+				xref: ["epub-33","html"],
```

Comment:
See my remark above. If the web platform is default, then we may not need it at all; otherwise using the web platform profile may be more future proof.

Reply:
This is what I used in the other

```
				xref: {
					profile: "web-platform",
					specs: ["epub-rs-33","epub-33"]
				},
```

and I did get references resolved for `url` for example... that is all I know!

Things are indeed a bit uneven, though, sometimes a bit mysterious...

Reply:
Ya, the docs could use some updating in that case. The only place I found the `profile` property is in [example 90](https://respec.org/docs/#example-specify-profile-and-use-additional-specs-for-searching). All the rest show "web-platform" defined with `xref`.

Reply:
I think what happens is that respec also tries to do some contextual analysis. That is why you see, eg, wcag2 on the list of the error report, but nothing about the URL spec...